keepassxc-browser/_locales/en/messages.json

```diff
@@ -312,9 +312,13 @@
         "description": "A popup message shown choosing what group user wants to use for new credentials."
     },
     "popupRememberErrorDefaultGroupNotFound": {
-        "message": "Error: Specified default group not found.",
+        "message": "Error: Specified default group not found. Creating all necessary groups.",
```

Comment:
This is not an error in this case, just an info. Should be reworded as: "Info: Default group created in connected database"